frontend/src/scenes/sceneLogic.js

```diff
@@ -124,7 +124,7 @@ export const sceneLogic = kea({
     },
     listeners: ({ values, actions }) => ({
         setScene: () => {
-            window.posthog && window.posthog.capture('$pageview')
+            window.posthog && window.posthog?.capture('$pageview')
```

Comment:
I think `window.posthog &&` would be extra now as you have the `?`

Reply:
Correct, removed that.